playbooks/02 - Use Case - CICD/Create a Request (Issue).json

```diff
@@ -258,7 +262,7 @@
                                 "defaultValue": null,
                                 "_previousName": "",
                                 "playbookField": true,
-                                "displayTemplate": "{{ firstname }} {{ lastname }}",
+                                "displayTemplate": "%7B%7B%20firstname%20%7D%7D%20%7B%7B%20lastname%20%7D%7D",
```

Comment:
undo this

Reply:
No .. due to this we are not able to see the username in dropdown list